### ISSUE TITLE
Aliases in special section

### ISSUE THREAD

**Issue by danebell** [clulab/reach#124]
Some papers, e.g. PMC2063868 and PMC2034365, have a special section naming their abbreviations. These should be exploited to inform the alias relations throughout the paper.

- The relations are very explicit, so we will benefit from relaxing our conservative requirement that both members be of the same label, and could also use the regularity to capture named entities such as simple chemicals that aren't being caught currently.
- Is it possible to ensure that these sections are read first?
- What are the titles that are used to name these sections in the nxml? So far: `Notes`, `Footnotes`, `Abbreviations used in this paper`, `List of abbreviations`, and one with no title, but interestingly with a footnote id that explains its contents in the nxml, namely `<fn-group content-type="abbreviations">`. This might also be exploited.

**Comment by enoriega** [comment]
Yes, we can read these sections before the rest to build the map of abbreviations. Are this titles (Notes, Footnotes, etc) attributes of a <sec> tag?

**Comment by danebell** [comment]
I've seen four strategies:

1. For Notes and Footnotes, it seems that they have dedicated tags `<notes>` in the former case and `<fn>` in the latter. As far as I've seen, `<fn>` is always within `<fn-group>` and sometimes within `<notes>` as in `<notes><fn-group><fn><p>Footnote here</p></fn></fn-group></notes>`. Sometimes this includes the def-list and def tags, as in `</fn-group><fn-group content-type="abbreviations"><fn id="FN2"><label>2</label><p>The abbreviations used are:
<def-list><def-item><term id="G1">Fc&#x003f5;R</term><def><p>Fc&#x003f5; receptor</p></def>...`

2. There's also the tag glossary, as in `<glossary><title>Abbreviations used in this paper</title><def-list><def-item><term>LAT</term><def><p>linker for activation of T cells</p></def><def-item></def-list></glossary>`, see PMC2193052

3. def-list is also used without the glossary or the footnote tag, as in `<def-list><title>Abbreviations used in this paper:</title><def-item><term>GAP</term><def><p>GTPase-activating protein</p></def>...`

4. Sometimes it's just a <sec> tag with an informative title including the lemma `abbreviation` or `glossary`, as in `<sec><title>List of abbreviations</title><p>Rb: Retinoblastoma; Cdk: Cyclin-dependent kinase;...`
